cfme/fixtures/ansible_fixtures.py

```diff
@@ -13,7 +13,8 @@
 @pytest.fixture(scope="module")
 def wait_for_ansible(appliance):
     appliance.server.settings.enable_server_roles("embedded_ansible")
-    appliance.wait_for_embedded_ansible()
```

Comment:
Assuming we have fixed the `is_embedded_ansible_running`, then I don't see the reason why to make the `appliance.wait_for_embedded_ansible()` called only when version <5.11

Reply:
This ties in with my comment here [0] - Agreed, put the version check within the method, and then consistently call it when needed and let it check various services based on appliance version.

[0] https://github.com/ManageIQ/integration_tests/pull/9203#discussion_r313868573

Reply:
@jaryn @mshriver Removed this check and added into method only as suggested :smile: